util/db/migrations/20220623082412-sfo-create-notes.js

```diff
@@ -0,0 +1,41 @@
+module.exports = {
+  // prettier-ignore
+  async up (queryInterface, Sequelize) {
+    await queryInterface.createTable('Notes', {
+      id: {
+        allowNull: false,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+        autoIncrement: true
+      },
+      ApplicationId: {
+        type: Sequelize.INTEGER,
+        references: {
+          model: 'Applications',
+          key: 'id'
+        }
+      },
+      note: {
+        type: Sequelize.TEXT
+      },
+      createdBy: {
+        type: Sequelize.STRING
+      },
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      },
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      },
+      deletedAt: {
+        type: Sequelize.DATE
+      }
+    });
+  },
+  // prettier-ignore
```

Comment:
Ditto.

Reply:
Done

Reply:
Resolved in https://github.com/Scottish-Natural-Heritage/standard-forestry-operations-api/pull/77/commits/e33b7df9f6afa48d4e7d2ef0bd9782a3a13aa654